packages/lit-html/package.json

```diff
@@ -58,7 +58,7 @@
     "dev": "scripts/dev.sh",
     "prepublishOnly": "npm run check-version",
     "test": "npm run test:dev && npm run test:prod",
-    "test:dev": "cd ../tests && npx wtr '../lit-html/development/**/*_test.(js|html)'",
+    "test:dev": "cd ../tests && npx wtr '../lit-html/development/**/lit-html_test.js'",
```

Comment:
Remove?

And note you can just do `npm run test:dev -- -- '../lit-html/development/**/lit-html_test.js'` on the command line to focus on one test if you don't want to use the CLI GUI for that.